src/System.Private.CoreLib/shared/System/Text/StringBuilder.cs

```diff
@@ -562,7 +562,7 @@ public int Length
         /// thus can be used in a C# 'foreach' statements to retreive the data in the StringBuilder
         /// as chunks (ReadOnlyMemory) of characters.  An example use is:
         /// 
-        ///      foreach (ReadOnlyMemory<char> chunk in sb.GetChunks())
+        ///      foreach (ReadOnlyMemory&lt;char&gt; chunk in sb.GetChunks())
```

Comment:
📝 This is going to be a poorly-formatted comment. Two primary changes that would be desired:

1. Wrap the paragraphs in `<para>…</para>` elements
2. For *blocks* of code (as opposed to inline code in a paragraph), use `<code>…</code>`